### DISCUSSION TITLE
lantern已经超过半年无法正常使用了，申请进行退款

### DISCUSSION THREAD

**Discussion by lydia7777777** [getlantern/lantern-forum-cn#947] [General]
从23年底gpt发布后至24年初，lantern短暂正常使用了几个月，随后无法试用gpt，进而google等访问也不稳定
期间提交过issue，但是提供log、更新版本仍然无法正常使用
故申请进行退款，望批准。

**Comment by lydia7777777** [comment]
我的邮箱：lydia_7777777@163.com
到期时间：2025/3/20

**Reply by MaxMKMax** [reply]
@lydia7777777 您好. 非常抱歉, 一般使用超过一个月了我们就无法退款了. 购买条款有写的. 对不起啊. 我们只能尽能力去修复问题. 请您帮我提交应用内报告然后给我 8 位数票号. 谢谢!

**Reply by lydia7777777** [reply]
![无标题](https://github.com/user-attachments/assets/513c5fcb-1329-499a-87dc-c5d3d6c24824)

不是，这还删帖么。 我真后悔一次性买了两年。
